### PR TITLE
Cha2251/file timestamp change

### PR DESCRIPTION
**Pull request by cha2251**
* When midi events are read in from a file, they now use relative timestamps starting at 0 for song start

### CODE REVIEW

**Comment by cha2251** [comment]
This was branched off of `cha2251/comm-overhaul`, so changed file count may be inaccurate. I blame GithHub for not letting me chain PRs

**Comment by cha2251** [comment]
Wait scratch that, it totally does. Kinda. Going to merge this one into the other one, then that one into main. #Science

File: src/user_interface/playing_page.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def __init__(self, output, song_name="DEFAULT"):
         ## vbox.addWidget()  ## ADD PROGRESS BAR WIDGET HERE to vbox
         ############################################################
 
-        vbox.addWidget(VisualizationWidget(parent=self, output=output))
+        #vbox.addWidget(VisualizationWidget(parent=self, output=output))
```
Comment (DavidTheFighter) [inline src/user_interface/playing_page.py:100]: Don't remove my visualizer!

**Review by DavidTheFighter — CHANGES_REQUESTED** [review]
ALGTM :D

File: src/user_interface/playing_page.py (hunk 1 above)
Comment (cha2251) [reply src/user_interface/playing_page.py:100]: Going to add a todo for the IPC swtichover

**Pull request merged**